Compile errors on zig 0.11.0 with zig build test

It looks like `TypeInfo` is now known as `Type` and has a few differences in fields

```test
╰─ zig build test
zig test util Debug native: error: the following command failed with 1 compilation errors:
/nix/store/6k2viwqb0dm38hijch7ibdp9sdwiz8h1-zig-0.11.0/bin/zig test /home/alex/workspace/mjoerussell/zig-odbc/src/util.zig -I/nix/store/ainxli0jmyfrznb2fczkaik7c31ch5in-unixODBC-2.3.12/include -L/nix/store/ainxli0jmyfrznb2fczkaik7c31ch5in-unixODBC-2.3.12/lib -lodbc -lc -fno-emit-bin --cache-dir /home/alex/workspace/mjoerussell/zig-odbc/zig-cache --global-cache-dir /home/alex/.cache/zig --name util --listen=-
Build Summary: 1/3 steps succeeded; 1 failed (disable with --summary none)
test transitive failure
└─ zig test util Debug native 1 errors
src/util.zig:2:29: error: root struct of file 'builtin' has no member named 'TypeInfo'
const TypeInfo = std.builtin.TypeInfo;
                 ~~~~~~~~~~~^~~~~~~~~
referenced by:
    Bitmask: src/util.zig:8:38
    test.bitmask: src/util.zig:74:22
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

```

--- a/src/util.zig
+++ b/src/util.zig
@@ -1,18 +1,18 @@
 const std = @import("std");
-const TypeInfo = std.builtin.TypeInfo;
+const Type = std.builtin.Type;
 
 pub fn Bitmask(comptime BackingType: type, comptime fields: anytype) type {
-    const BaseStruct = packed struct {};
-    var base_struct_info = @typeInfo(BaseStruct);
-
-    var incoming_fields: [fields.len]TypeInfo.StructField = undefined;
+    var incoming_fields: [fields.len]Type.StructField = undefined;
     inline for (fields, 0..) |field, i| {
-        incoming_fields[i] = .{ .name = field[0], .field_type = bool, .default_value = false, .is_comptime = false, .alignment = @alignOf(bool) };
+        incoming_fields[i] = .{ .name = field[0], .type = bool, .default_value = &false, .is_comptime = false, .alignment = @alignOf(bool) };
     }
 
-    base_struct_info.Struct.fields = incoming_fields[0..];
-
-    const BitmaskFields = @Type(base_struct_info);
+    const BitmaskFields = @Type(.{ .Struct = .{
+        .layout = .Auto,
+        .fields = incoming_fields[0..],
+        .decls = &[_]std.builtin.Type.Declaration{},
+        .is_tuple = false,
+    } });
 
     return struct {
         pub const Result = BitmaskFields;
@@ -49,12 +49,12 @@ pub fn Bitmask(comptime BackingType: type, comptime fields: anytype) type {
 pub fn EnumErrorSet(comptime BaseEnum: type) type {
     switch (@typeInfo(BaseEnum)) {
         .Enum => |info| {
-            var error_set: [info.fields.len]TypeInfo.Error = undefined;
+            var error_set: [info.fields.len]Type.Error = undefined;
             inline for (info.fields, 0..) |field, index| {
                 error_set[index] = .{ .name = field.name };
             }
 
-            return @Type(TypeInfo{ .ErrorSet = error_set[0..] });
+            return @Type(Type{ .ErrorSet = error_set[0..] });
         },
         else => @compileError("EnumErrorSet requires an enum, found " ++ @typeName(BaseEnum)),
     }

